data/resources/settings.json

```diff
@@ -21,24 +21,23 @@
     "ShowBuildingsInBlueprint": false,
     "ZoneLayerTransparency": 0.6
   },
-    "Graphics": {
-        "FullScreen": false,
-        "FullScreenMode": 0,
-        "Resolution": {
-            "Screen_Height": 600,
-            "Screen_Width": 800
-        },
-        "VSYNC": true
+  "Graphics": {
+    "FullScreen": false,
+    "FullScreenMode": 0,
+    "Resolution": {
+      "Screen_Height": 600,
+      "Screen_Width": 800
     },
-    "User Interface": {
+    "VSYNC": true
+  },
+  "User Interface": {
     "BuildMenuPosition": "BOTTOM",
     "FontFilename": "resources/fonts/arcadeclassics.ttf",
-        "Language": "en",
-        "SubMenuButtonHeight": 32,
-        "SubMenuButtonWidth": 32
-    },
-    "Debug": {
-      "WriteErrorLogToFile": false
-
-    }
-}
+    "Language": "en",
+    "SubMenuButtonHeight": 32,
+    "SubMenuButtonWidth": 32
+  },
+  "Debug": {
+    "WriteErrorLogToFile": false
+  }
```

Comment:
```suggestion
  }
    "Font":{
      "FontFilename": "resources/fonts/arcadeclassics.ttf",
      "FontSize": 20
      }
```

Reply:
ok, my example doesn't seem quite right, but add a font category with filename and fontsize.

Reply:
Side note: use the font from master, i changed the default font to another font that can display .